package-lock.json

```diff
@@ -3865,120 +3865,121 @@
       }
     },
     "d3": {
```

Comment:
Please drop the changes here as this should be pushed within the release commit for easy tracking (e.g. [commit for release 2.5.0](https://github.com/danielcaldas/react-d3-graph/commit/b476ef5bfff01a15813a201d2f08d450d5e288ae))